docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/index.md

```diff
@@ -0,0 +1,205 @@
+| f9e6fa53-61f4-45f4-adb1-b75e018f8a32Direct0         | f9e6fa53-61f4-45f4-adb1-b75e018f8a32 | f0009028775170427694 | 2023-06-11 15:33:03.000 | 20.42            | Direct                           | Direct         | 0            | 1           | 20.42                   | 20.42                  | 20.42              | 20.42                      |
+| c3bf5de5-20c7-42d6-9543-afd9ecf133f5Video0          | c3bf5de5-20c7-42d6-9543-afd9ecf133f5 | f0008284662789123943 | 2023-07-07 13:05:55.000 | 200              | Video                            | Video          | 0            | 1           | 200                     | 200                    | 200                | 200                        |
+| 1487c09f-f17f-4c76-990e-55c46f8f621cDisplay_Other0  | 1487c09f-f17f-4c76-990e-55c46f8f621c | f0006911334202687206 | 2023-07-19 04:27:51.000 | 66.5             | Display_Other > Organic_Search   | Display_Other  | 0            | 2           | 66.5                    | 0                      | 33.
+
+3. The `derived.snowplow_attribution_campaign_attributions` does the same, only for campaigns not channels.
+
+### Drop and recompute reporting tables/views:
+
+1. The `derived.snowplow_attribution_path_summary` shows the campaign/channel paths and the assiciated conversions (an optionally non-conversions, if the `path_to_non_conversions` table is enabled through its related variable `enable_path_to_non_conversions`)
+
+| TRANSFORMED_PATH                   | CONVERSIONS | NON_CONVERSIONS | REVENUE  |
+|------------------------------------|-------------|-----------------|----------|
+| Direct                             | 3           | 25              | 94.42    |
+| Organic_Search                     | 2           | 26              | 206.5    |
+| Referral > Direct                  | 0           | 1               | 0        |
+| Video                              | 1           | 2               | 200      |
+| Organic_Search > Paid_Search_Other | 0           | 2               | 0        |
+
+
+2. The view called `derived.snowplow_attribution_overview` is tied to a dispatch macro of the same name which lets you overwrite in your proects macros, if needed. Given you specify your `var('snowplow__spend_source')` it will calculate the ROAS for you for each channel and campaign:
+
+
+| PATH_TYPE | ATTRIBUTION_TYPE | TOUCH_POINT| IN_N_CONVERSION_PATHS | ATTRIBUTED_CONVERSIONS | MIN_CV_TSTAMP | MAX_CV_TSTAMP | SPEND | SUM_CV_TOTAL_REVENUE | ATTRIBUTED_REVENUE | ROAS |
+|----------|-------------|----------------|---|----------------|-------------------------|-------------------------|--------|------------|------------------|----------------|
+| campaign | first_touch | Campaign 2     | 2 | 1.000000000000 | 2023-07-19 04:27:51.000 | 2023-07-25 07:52:34.000 | 100000 | 206.500000 | 206.500000000000 | 0.002065000000 |
```

Comment:
Could we round off some of these values , like attributed_conversions, for easier readability